UberSignals/UBSignal.h

```diff
@@ -120,6 +120,11 @@ typedef void (^UBSignalObserverChange)(UBSignalObserver *signalObserver);
  */
 @property (nonatomic, strong) UBSignalObserverChange observerRemoved;
 
+/**
+ Helper constructor, Initializes an empty signal with the EmptySignal protocol.
```

Comment:
Not a constructor, but a "Helper factory method, constructs a Signal instance with the EmptySignal protocol".